pyccel/naming/cnameclashchecker.py

```diff
@@ -33,7 +33,7 @@ class CNameClashChecker(metaclass = Singleton):
         'GET_INDEX_FUNC_H2', 'GET_INDEX_FUNC', 'GET_INDEX',
         'INDEX', 'GET_ELEMENT', 'free_array', 'free_pointer',
         'get_index', 'numpy_to_ndarray_strides',
-        'numpy_to_ndarray_shape', 'get_size'])
+        'numpy_to_ndarray_shape', 'get_size', 'order_f', 'order_c'])
```

Comment:
can you add `array_copy_data` and the other functions that are added to the internal C library